muss/channels.py

```diff
@@ -0,0 +1,68 @@
+class Channel(object):
+    def __init__(self, name):
+        if name in _channels:
+            raise ValueError("There's already a channel named {}.".format(name))
+        _channels[name] = self
+        self.name = name
```

Comment:
Do we want to constrain channel names? For example, are they allowed to have spaces in them?